nemo/collections/nlp/modules/common/tokenizer_utils.py

```diff
@@ -82,7 +82,7 @@ def get_tokenizer(
 
     if tokenizer_name == 'sentencepiece':
         return nemo.collections.common.tokenizers.sentencepiece_tokenizer.SentencePieceTokenizer(
-            model_path=tokenizer_model, special_tokens=special_tokens
+            model_path=tokenizer_model, special_tokens=special_tokens, legacy=True
```

Comment:
could you add docstring for bpe_dropout?

Reply:
Added.